### PR TITLE
Add collections

### PR DESCRIPTION
**Pull request by emilyville**
Add some collections! 

woo woooooooo

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
 description: > # this means to ignore newlines until "baseurl:"
-  Write an awesome description for your new site here. You can edit this
-  line in _config.yml. It will appear in your document head meta (for
-  Google search results) and in your feed.xml site description.
+  Description will go here.
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "http://yourdomain.com" # the base hostname & protocol for your site
 twitter_username: jekyllrb
-github_username:  jekyll
+github_username: jekyll
```
Comment (awong-dev) [inline _config.yml]: what are these for? Should we remove them?

**Comment by emilyville** [comment]
fixes issue #47

**Comment by awong-dev** [comment]
LGTM w/ 1 comment.

Feel free to fix in a separate PR.

File: _config.yml (hunk 1 above)
Comment (emilyville) [inline _config.yml]: yeah, probably. It's a variable in the footer so would be good to remove when we make our footer.

**Comment by awong-dev** [comment]
squished commits for @emilyville. Will merge after travis completes.

**Pull request merged**